tgui/packages/tgui/interfaces/LibraryScanner.js

```diff
@@ -0,0 +1,79 @@
+import { useBackend } from '../backend';
+import { Button, Flex, NoticeBox, Section } from '../components';
+import { Window } from '../layouts';
+
+export const LibraryScanner = (props, context) => {
+  return (
+    <Window
+      title="Library Scanner"
+      width={350}
+      height={150}>
+      <BookScanning />
+    </Window>
+  );
+};
+
+export const BookScanning = (props, context) => {
```

Comment:
```suggestion
const BookScanning = (props, context) => {
```
There is no need to export these components if you're only using them here